run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+curl -L "https://github.com/fpco/amber/releases/download/v0.1.5/amber-x86_64-unknown-linux-musl" -o "amber"
```

Comment:
Only question is here. Would it make sense to include the amber exe in the docker image instead?

Reply:
Good point, moved to the docker image.

Reply:
Good point, moved to the docker image instead.